browser/ui/webui/brave_webui_source.cc

```diff
@@ -233,19 +233,26 @@ void CustomizeWebUIHTMLSource(content::WebUI* web_ui,
         { "braveNewsChannel-Crypto", IDS_BRAVE_NEWS_CHANNEL_CRYPTO},
         { "braveNewsChannel-Culture", IDS_BRAVE_NEWS_CHANNEL_CULTURE},
         { "braveNewsChannel-Entertainment", IDS_BRAVE_NEWS_CHANNEL_ENTERTAINMENT}, // NOLINT
+        { "braveNewsChannel-Entertainment News", IDS_BRAVE_NEWS_CHANNEL_ENTERTAINMENT_NEWS}, // NOLINT
```

Comment:
Curious the strings that have space works?

Reply:
It's just a key in a JavaScript object :)